src/lib/logs/log_self_monitoring.py

```diff
@@ -25,7 +25,7 @@
 from lib.context import LoggingContext, LogsSfmContext, DynatraceConnectivity, LogsContext
 from lib.credentials import create_token, get_dynatrace_log_ingest_url_from_env
 from lib.instance_metadata import InstanceMetadata
-from lib.logs.log_forwarder_variables import LOGS_SUBSCRIPTION_PROJECT, LOGS_SUBSCRIPTION_ID, \
+from lib.logs.log_forwarder_variables import GCP_PROJECT, LOGS_SUBSCRIPTION_ID, \
```

Comment:
In case if anybody already used our Helm template and pulls new docker image, we should fallback to `LOGS_SUBSCRIPTION_PROJECT` if `GCP_PROJECT` is not set